index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+    <header>
+        <span class="logo-placeholder">
+            Kenny Salazar
+        </span>
+        <div class="menu-icon">
+        </div>
+    </header>
+    <section class="headline">
+        <h1>Hey There.<br>I'm Kenny</h1>
+        <h2>I’am a Software Developer</h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and 
+            have a project you need coded, don’t hestiate to contact me.
+        </p>
```

Comment:
Fantastic job! Well done implementing this section's elements with the proper tags 😎